src/Arcus.Observability.Tests.Unit/Serilog/ApplicationInsightsTelemetryConverterTests.cs

```diff
@@ -699,6 +697,37 @@ public void LogEvent_WithEvent_CreatesEventTelemetry()
             });
         }
 
+        [Fact]
+        public void LogException_WithException_CreatesExceptionTelemetry()
+        {
+            // Arrange
+            const string eventName = "Order Invoiced";
+            string platform = $"platform-id-{Guid.NewGuid()}";
+            var exception = new PlatformNotSupportedException(platform);
+            var spySink = new InMemoryLogSink();
+            string operationId = $"operation-id-{Guid.NewGuid()}";
+            ILogger logger = CreateLogger(spySink, config => config.Enrich.WithProperty(ContextProperties.Correlation.OperationId, operationId));
+
+            logger.LogCritical(exception, exception.Message);
+            LogEvent logEvent = Assert.Single(spySink.CurrentLogEmits);
+            Assert.NotNull(logEvent);
+
+            var converter = ApplicationInsightsTelemetryConverter.Create();
+
+            // Act
+            IEnumerable<ITelemetry> telemetries = converter.Convert(logEvent, formatProvider: null);
+
+            // Assert
+            Assert.Collection(telemetries, telemetry =>
+            {
+                var exceptionTelemetryType = Assert.IsType<ExceptionTelemetry>(telemetry);
```

Comment:
Glad you're using the full functionality of assertions! 😊

Reply:
Because you've made it so simple!